README.rst

```diff
@@ -69,6 +69,15 @@ You can specify the base URL using a `configuration file`_:
   [pytest]
   base_url = http://www.example.com
 
+Using an environment variable
```

Comment:
Nit: For consistency with the other headings, can you title case 'Environment Variable'